styles/sections/_faqs.scss

```diff
@@ -0,0 +1,70 @@
+@use '../colors';
+
+.faq-list {
+  &:first-of-type {
+    @container body (min-width: 1024px) {
+      display: block;
+    }
+  }
+
+  li {
+    text-align: left;
+    border-bottom: 1px solid;
+    padding: 24px 0;
+
+    .h4,
+    h3 {
+      padding: 0 32px 0 0;
+      margin-bottom: 0;
+      text-transform: none;
+
+      @container body (min-width: 768px) {
+        padding-right: 48px;
+      }
+    }
+
+    .faq-details {
+      position: relative;
+
+      &:focus-within {
+        outline: 2px dashed colors.$color-white;
+        outline-offset: 12px;
+      }
```

Comment:
brainstorming here, i know you were stuck on this so we can move on on if this idea doesn't work - how about moving these styles to the `summary` element below but with the `focus-visible` style?